nalgebra-sparse/tests/unit_tests/mod.rs

```diff
@@ -3,6 +3,8 @@ mod convert_serial;
 mod coo;
 mod csc;
 mod csr;
+#[cfg(feature = "io")]
```

Comment:
Should make sure to run the IO tests on CI!

Reply:
should modify something inside `workflows/nalgebra-ci-build.yml` ?

because I never tried about github CI before, so don't know about it

Reply:
I can handle this part!